server/app/routes/api/v1/va/ambassadors.js

```diff
@@ -225,24 +246,46 @@ async function disapproveAmbassador(req, res) {
  *
  */
 async function makeAdmin(req, res) {
-  let found = await req.neode.first('Ambassador', 'id', req.params.ambassadorId);
+  let found = await req.neode.first('Ambassador', 'id', req.params.ambassadorId)
 
   if (!found) {
-    return error(404, res, "Ambassador not found");
+    return error(404, res, 'Ambassador not found')
   }
 
-  let json = {...{admin: true}};
-  await found.update(json);
+  let json = {...{admin: true}}
+  await found.update(json)
 
   // send email in the background
-  setTimeout(async ()=> {
-    let address = JSON.parse(found.get('address'));
-    let body = makeAdminEmail(found, address);
-    let subject = `New Admin for ${ov_config.organization_name}`;
-    await mail(ov_config.admin_emails, null, null, subject, body);
-  }, 100);
-
-  return _204(res);
+  setTimeout(async () => {
+    let address = JSON.parse(found.get('address'))
+    let body = makeAdminEmail(found, address)
+    let subject = `New Admin for ${ov_config.organization_name}`
+    await mail(ov_config.admin_emails, null, null, subject, body)
+  }, 100)
+
+  return _204(res)
+}
+
+/*
+ *
+ * updateAmbassdaorW9status(req, res)
+ *
+ * This function makes an admin out of an Ambassador. Ambassadors are listed in the admin panel,
```

Comment:
I think something got a little mixed up here maybe?

I see `updateW9Ambassdaor` defined above, and it looks almost identical?

Reply:
removed